### DISCUSSION TITLE
What is the name of using monospaced fonts?

### DISCUSSION THREAD

**Discussion by 4SM** [black7375/Firefox-UI-Fix#563] [Q&A] [answered]
`userChrome.theme.monospace:true`

very nice

![image](https://user-images.githubusercontent.com/25581533/210785750-37c354ca-dc41-4ee6-b16c-c0c3be3d024c.png)

**Comment by black7375** [comment, marked as answer]
Is it your computer?

Check the font information at `about:preferences`.
![image](https://user-images.githubusercontent.com/25581533/210786480-d2fef954-10c1-4ba2-86e0-df0dc5cea8be.png)
![image](https://user-images.githubusercontent.com/25581533/210786622-6b4aad84-070b-4718-a4f7-d0bdbe6801e5.png)
